mango_time_series/mango/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+Main file
+"""
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+__version__ = "0.2.3a1"
```

Comment:
Wrong version

Reply:
Could add a test like this
```python
import mango
from unittest import TestCase
import os
from mango.tests.const import normalize_path


class InitializeTest(TestCase):
    def test_version_matches(self):
        with open(normalize_path(os.path.join("..","..","pyproject.toml")),"r") as f:
            pyproject = f.read()
        version = pyproject.split("version = ")[1].split("\n")[0].replace('"','')
        self.assertEqual(mango.__version__,version)
```

However, this raises the following question. What would happen when we run ```mango.__version__``` will it print base ```__version__``` or time_series ```__version__```